cmd/engine/.dagger/main.go

```diff
@@ -232,11 +232,13 @@ var targets = []struct {
 	GPUSupport bool
 }{
 	{
-		Name:      "alpine (default)",
-		Tag:       "%s",
-		Image:     DistroAlpine,
-		Platforms: []dagger.Platform{"linux/amd64", "linux/arm64"},
+		Name:  "alpine (default)",
+		Tag:   "%s",
+		Image: DistroAlpine,
+		// Platforms: []dagger.Platform{"linux/amd64", "linux/arm64"},
```

Comment:
note to self to rm before merge (makes publishing custom engine image tolerable)